src/shruti/FormData.java

```diff
@@ -0,0 +1,68 @@
+package shruti;
+
+	
+	import org.openqa.selenium.By;
+	import org.openqa.selenium.WebDriver;
+	import org.openqa.selenium.WebElement;
+	import org.openqa.selenium.chrome.ChromeDriver;
+
+	public class FormData {
+
+		// WebDriver driver = new ChromeDriver();
+
+		void dataEntryinForm() {
+			System.setProperty("webdriver.chrome.driver",
+					".\\Resources\\chromedriver.exe");
+			WebDriver driver = new ChromeDriver();
+			driver.get("file:///D:/Technocredit/Selenium%20recordings/@Selenium/SeleniumAssignment_1.html");
+			WebElement firstName = driver.findElement(By.id("first name"));
+			firstName.sendKeys("Shruti");
+			WebElement lastName = driver.findElement(By.id("last name"));
+			lastName.sendKeys("Dubey");
+			WebElement email = driver.findElement(By.id("E-mail"));
```

Comment:
You should only store any value in webElement when you want to perform more than 1 action on it..
But here you are only performing one action which sendKeys() so you should write like driver.findElement(By.id("E-mail")).sendKeys("shrutidubey@gmail.com");

Reply:
changes done..Thanks